middleware/varnish/README.md

```diff
@@ -0,0 +1,57 @@
+# MIDDLEWARE Varnish SignalFx detectors
+
+## Prerequisites
+
+This monitor uses the **varnishstat** command. The **signalfx-agent** user must be
+able to use this command. In order to do that, you need to start **varnishncsa**.
+On **Varnish 4**, you need to add in the **/etc/default/varnishncsa** file :
+
+```hcl
+VARNISHNCSA_ENABLED=1
+```
+
+Following by :
+
+```hcl
+systemctl start varnishncsa.service
+```
+
+Then, you need to add the **signalfx-agent** user to the varnish group :
+
+```hcl
```

Comment:
again not hcl and I think it is probably better to forward to official telegraf doc : https://github.com/influxdata/telegraf/blob/master/plugins/inputs/varnish/README.md#permissions